DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/CourseSetup/CourseSetupControllerTests.cs

```diff
@@ -151,22 +219,446 @@ public void Index_with_CLEAR_filterBy_and_new_filter_query_parameter_sets_new_co
 
         [Test]
         public void Index_with_no_filtering_should_default_to_Active_courses()
+        {
+            // When
+            var result = controller.Index();
+
+            // Then
+            result.As<ViewResult>().Model.As<CourseSetupViewModel>().FilterBy.Should()
+                .Be("Status|Active|true");
+        }
+
+        [Test]
+        public void AddCourseNew_sets_new_temp_data()
+        {
+            // When
+            var result = controller.AddCourseNew();
+
+            // Then
+            controller.TempData.Peek<AddNewCentreCourseData>().Should().NotBeNull();

```

Comment:
Wrapping multiple assertions in an `AssertionScope` will allow us to capture all potential issues rather than just the first.

Reply:
added AssertionScope to all the tests in this file where it needs it